### PR TITLE
APPS-803 Sonatype Nexus publishing setup readme.

### PR DESCRIPTION
**Pull request by r-i-v-a**

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -60,11 +107,10 @@ When a PR is merged into `develop`, SNAPSHOT packages are automatically publishe
 
 ### Releasing to Maven
 
-Note: this process is currently coordinated by John Didion - please request from him a release of the updated project(s).
+Note: this process is currently coordinated by Riva Nathans & John Didion -- please contact them to request a release of the updated library(ies).
 
-1. From the release branch, run `sbt 'project <name>' publishSigned -DreleaseTarget=sonatype` for each project being released.
-   - You will need to have the SonaType PGP private key on your machine, and you will need the password.
-2. Go to [nexus repository manager](https://oss.sonatype.org/#stagingRepositories), log in, and go to "Staging Repositories".
+1. From the release branch, run `sbt 'project <name>' publishSigned -DreleaseTarget=sonatype` for each project being released. You will need to have completed the "Sonatype Nexus publishing setup" instructions above. You will be prompted to enter your key's passphrase.
```
Comment (commandlinegirl) [inline DEVELOPING.md:114]: Do we have a GitHub action workflow for this (or are planning to add)?

**Review by jdidion — DISMISSED** [review]

File: DEVELOPING.md [hunk 2]
```diff
@@ -41,6 +41,53 @@ When a PR is merged into `develop`, SNAPSHOT packages are automatically publishe
 
 ## Releasing
 
+### Sonatype Nexus publishing setup
```
Comment (commandlinegirl) [inline DEVELOPING.md:44]: Please check my comments in https://github.com/dnanexus-rnd/wdlTools/pull/178 for review. Thanks!

**Review by commandlinegirl — CHANGES_REQUESTED** [review]

**Comment by r-i-v-a** [comment]
From discussion, agreed to merge documentation of current process

Jiras to publish to GitHub packages https://jira.internal.dnanexus.com/browse/APPS-809 , https://jira.internal.dnanexus.com/browse/APPS-808

**Review by commandlinegirl — APPROVED** [review]

File: DEVELOPING.md [hunk 3]
```diff
@@ -41,6 +41,53 @@ When a PR is merged into `develop`, SNAPSHOT packages are automatically publishe
...
+
+- Have `dnanexus`'s password for [Sonatype Nexus repository manager](https://oss.sonatype.org/#stagingRepositories). Riva Nathans and John Didion have the password.
```
Comment (commandlinegirl) [inline DEVELOPING.md]:
```suggestion

We use [Sonatype Nexus repository manager](https://oss.sonatype.org/#stagingRepositories) to release artifacts. In order to set it up:

- Have `dnanexus`'s password for Sonatype. Riva Nathans and John Didion have the password.
```

**Review by commandlinegirl — DISMISSED** [review]

**Review by commandlinegirl — APPROVED** [review]
LGTM, thank you, Riva!

**Pull request merged**